lib/oxidized/model/hpvc.rb

```diff
@@ -0,0 +1,22 @@
+class Hpvc < Oxidized::Model
+  # HPE Moonshot Switch / HP Virtual Connect Linux
+  # sometimes the prompt might have a leading nul or trailing ASCII Bell (^G)
+  prompt /^\0*(\([\w.-]+\)).?[>#]$/
+  comment '# '
+
+  cfg :telnet, :ssh do
+    post_login 'enable'
```

Comment:
Do these devices never require enable password? Should this perhaps be wrapped around in a conditional block depending on if an enable variable is set, and if it contains a password string?